pages/osx/krunvm.md

```diff
@@ -0,0 +1,24 @@
+# krunvm
+
+> CLI-based utility for creating microVMs from OCI images.
+> More information: <https://github.com/containers/krunvm>.
+
+- Create MicroVM based on Fedora:
+
+`krunvm create docker.io/fedora --cpus {{number of vcpus}} --mem {{Memory in mbs}} --name {{name}}`
+
+- Start a MicroVM created:
```

Comment:
```suggestion
- Start a specific image:
```

- Optional fix: `a specific` may sound better.
- Optional fix: `MicroVM` can be implicitly supposed to be a type of OCI images.